pkg/kubernetes/utils.go

```diff
@@ -127,12 +129,20 @@ func (utilsClient *UtilsClient) DrainNCN(ncn string) error {
 	utilsClient.Logger.debugf("Pod disruption budgets satisfied.")
 
 	// Now finally we can drain the node.
-	err = utilsClient.runNodeDrain(utilsClient.helper, ncn)
-	if err != nil {
-		return fmt.Errorf("failed to drain NCN: %w", err)
+	attempts := 3
+	sleep, _ := time.ParseDuration("2s")
```

Comment:
Should `"2s"` be replaced with `attempts-1` with an "s" appended to it? then changing `attempts` wouldn't require changing `sleep`.

Reply:
i see what you mean, just have a control over total time? but i don't think we care the relation between attempts and sleep. Depends on our test, we might change it to 10s but try only twice